javascript/new-src/src/css/style.css

```diff
@@ -217,3 +217,28 @@ div#header-universe > div{
 .settings, .input-group, .save-settings {
  margin-top: 5px;
 }
+
+/* Custom checkboxes for plugins page */
```

Comment:
I've historically found custom pointer support a bit lacking generally, I think Chrome was the only one that did it. Although from memory that was a completely custom .ico pointer.